app/models/site_config.rb

```diff
@@ -23,7 +23,7 @@ class SiteConfig < RailsSettings::Base
 
   # Authentication
   field :allow_email_password_registration, type: :boolean, default: false
-  field :allow_email_password_login, type: :boolean, default: false
+  field :allow_email_password_login, type: :boolean, default: true
```

Comment:
is [this description](https://github.com/forem/forem/blob/7ce3b8abd25fb2424e67b07d18d11d9676e1b2a5/app/lib/constants/site_config.rb#L9) still correct? if we default it to being on/`true`, then we're adding it as an additional auth option, but not the _only_ option, right?

wondering if we should update the wording there because someone might be confused why it defaults to being on but other auth providers also show up when you go to sign in

Reply:
I'll revisit the functionality to make sure I understand it all in the morning.

Reply:
Okay I looked at this and I think this makes sense and is straightforward. I updated the copy in the description.